web-console/future-openshift/health-checks/health-checks.md

```diff
@@ -0,0 +1,21 @@
+# Health Checks
+
+Health checks allow users to monitor various container states. The following design adds the ability to create and edit health checks via forms.
+
+## Design
+* To access health checks, users can drill down into a container from the containers list on any resource overview tab.
+![overview](img/C1.png)
+
+* On the container details page, there will be an editable Health Checks section. There are 3 types of probes users can add; each probe will be listed in the section so that users can quickly see any probes they have created or see which ones they still need to create.
+![container-overview](img/C2.png)
+
+* The health check form allows users to add a single probe of each type. Users can add any combination of probes they wish. Probes can be removed by clicking "Remove [Probe Type]".
+![form](img/C3.png)
+
+* The form fields for each probe type are the same.
+* Note that probe forms will be read-only for some resources, similar to environments. In these cases, an inline alert will redirect users to the parent resource.
+![form-long](img/C4.png)
```

Comment:
I'm curious about how this design where the help text is below is different from the recent pattern we've seen where the field label is underlined and the user gets more details in a popover. Should we do the same here?

Reply:
I'd probably stick with the [helper text pattern](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/forms) that PatternFly recommends for these form fields because they provide "information that a user needs to know in order to enter the correct information". The very new (and unproven and likely less discoverable) underline pattern is probably most similar to the infotip pattern that's used "for context or background information that is "nice to have"."